packages/forms-web-app/src/routes/final-comment/final-comment.js

```diff
@@ -14,5 +15,6 @@ router.post(
 	validationErrorHandler,
 	finalCommentController.postAddFinalComment
 );
+router.get('/final-comment-submitted', finalCommentSubmittedController.getFinalCommentSubmitted);
```

Comment:
should this be in its own router file `/routes/final-comment/final-comment-submitted`?

Reply:
done